waspc/src/Wasp/Generator/Crud.hs

```diff
@@ -20,18 +20,18 @@ import qualified Wasp.AppSpec as AS
 import qualified Wasp.AppSpec.Crud as AS.Crud
 import Wasp.Generator.Common (makeJsArrayFromHaskellList)
 import qualified Wasp.Generator.Crud.Routes as Routes
-import qualified Wasp.Psl.Ast.Model as PslModel
+import qualified Wasp.Psl.Ast.Schema as Psl.Ast
 import qualified Wasp.Util as Util
 
-getCrudOperationJson :: String -> AS.Crud.Crud -> PslModel.Field -> Aeson.Value
+getCrudOperationJson :: String -> AS.Crud.Crud -> Psl.Ast.Field -> Aeson.Value
```

Comment:
This read nicely because we only concerned ourselves with PSL models and not the full PSL. So the qualified import of `PslModel` made sense where now it no longer makes sense since we are concerning ourselves with the full PSL.

I can rename the `Field` to `ModelField` so it's obvious it's specific to model.

Reply:
Ok, great!